src/engine/test/source/builder/builders/opBuilderHelperNetInfoAddress_test.cpp

```diff
@@ -0,0 +1,615 @@
+/* Copyright (C) 2015-2022, Wazuh Inc.
+ * All rights reserved.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+#include "opBuilderHelperNetInfoAddress.hpp"
+
+#include <any>
+#include <gtest/gtest.h>
+#include <thread>
+#include <vector>
+
+#include <baseTypes.hpp>
+#include <wdb/wdb.hpp>
+
+#include "socketAuxiliarFunctions.hpp"
+
+using namespace base;
+namespace bld = builder::internals::builders;
```

Comment:
```suggestion
using builder::internals::builders;
```
And please remove every `bld::` occurrences.